runtime/lua/vim/treesitter/language.lua

```diff
@@ -0,0 +1,26 @@
+local a = vim.api
+
+local M = {}
+
+function M.require_language(lang, path)
+  if vim._ts_has_language(lang) then
+    return true
+  end
+  if path == nil then
+    local fname = 'parser/' .. lang .. '.*'
```

Comment:
I thought these had to have `.so` endings?

Reply:
Not sure about that part... I think both `.o` and `.so` work.

Reply:
Doesn't it depend on the platform? Unix-like systems will probably be one of those, but Windows and maybe Mac will be different.

Reply:
OHh.... will this make `c` match `cpp`?

Reply:
Oh that's a good point.

Reply:
It should probably be like `string.format("parser/%s\\..*", lang)`  or similar. It should be <language>.<extension>

Reply:
Confirmed to be in its correct form, this wil be expanded to 'parser/{mylang}.*` and at some point will be passed to `gen_expand_wildcards()` which only consider `*` as wildcards.